app/styles/partials/overrides.scss

```diff
@@ -84,7 +84,7 @@ body.dimmable.undetached.dimmed {
 }
 
 .space-between {
-  justify-content: space-between;
+  justify-content: space-evenly;
```

Comment:
You can't just change the class content without impacting everywhere the class is used

Reply:
i will fixed this